### PR TITLE
docs: Remove yay helper comment

### PR DESCRIPTION
**Pull request by riusricardo**
Remove old comment.

### CODE REVIEW

**Comment by riusricardo** [comment]
@shawntabrizi Developing on Ink! will require "wasm2wat" and "wat2wasm" in the long term? Or it is just for now?
Just asking because of the missing packages on Debian based distros.
Only because of this I enabled an alternative install method [Parity Scripts #81](https://github.com/paritytech/scripts/pull/81)

Is this something needed or it will be replaced in the near future?

**Comment by shawntabrizi** [comment]
@pepyakin can maybe speak more towards the long term need for wasm2wat and wat2wasm, but in general, we need it for this step: (learn more at the bottom)

https://github.com/substrate-developer-hub/substrate-contracts-workshop/blob/master/0/building-your-contract.md

> TL;DR, this line is adding a maximum size to the Wasm memory declaration, which by default is not included.

**Comment by pepyakin** [comment]
yup these tools there are a temporary hack

**Comment by riusricardo** [comment]
@pepyakin @shawntabrizi Thanks! I'll close the script PR. It will not be valid for future development. 
However, I'll keep it on my fork because it is useful for workshops and actual development.
